index.php

```diff
@@ -28,10 +28,18 @@
     <a href="./admin/index.html">Admin Page</a>
     <div class="navbar">
       <ul>
+        <li class="list" data-category="All">
+          <a href="#">
+            <!-- use icons object -->
+            <span class="icon"><i class="fa-solid fa-earth-americas"></i></span>
+            <span class="text">All</span>
+          </a>
+        </li>
         <?php
 
         // create icons object to translate "database name" => "icon name"
         $icons = [
+          "All" => 'earth-americas',
```

Comment:
No need to put it into the array. The array is used for the categories in the database :)

Reply:
thanks for the correction👍 and ' is also good for me.